isis/tests/FunctionalTestsPointreg.cpp

```diff
@@ -0,0 +1,377 @@
+  ASSERT_EQ(int(measures["Registered"]), 24);
+  
+  // Check flatFile
+  QFile flatFile(flatFilePath);
+  ASSERT_TRUE(flatFile.size() > 0) ;
+
+  // Check output control network
+  ControlNet outNet(outNetPath);
+  ASSERT_EQ(int(outNet.GetNumPoints()), 15);
+  ASSERT_EQ(int(outNet.GetNumMeasures()), 39);
+}
+
+TEST_F(ThreeImageNetwork, FunctionalTestPointregOutputOptions) {
+  // Populate cubes
+  LineManager line(*cube1);
+  double pixelValue = 0.0;
+  for(line.begin(); !line.end(); line++) {
+    for(int i = 0; i < line.size(); i++) {
+      line[i] = pixelValue++;
+    }
+    cube1->write(line);
+    cube2->write(line);
+    cube3->write(line);
+  }
+  cube1->close();
+  cube2->close();
+  cube3->close();
+
+  QTemporaryDir prefix;
+  QString flatFilePath = prefix.path() + "/flatfile.csv";
+  QString outNetPath = prefix.path() + "/outNet.net";
+  QVector<QString> argsA = { "fromlist=" + cubeListFile,
+                            "cnet=data/threeImageNetwork/controlnetwork.net",
+                            "deffile=data/threeImageNetwork/autoRegTemplate.def",
+                            "flatfile=" + flatFilePath,
+                            "onet=" + outNetPath,
+                            "outputfailed=no",
+                            "points=ignored" };
+  QVector<QString> argsB = { "fromlist=" + cubeListFile,
+                            "cnet=data/threeImageNetwork/controlnetwork.net",
+                            "deffile=data/threeImageNetwork/autoRegTemplate.def",
+                            "flatfile=" + flatFilePath,
+                            "onet=" + outNetPath,
+                            "measures=candidates",
+                            "points=ignored" };
+  QVector<QString> argsC = { "fromlist=" + cubeListFile,
+                            "cnet=data/threeImageNetwork/controlnetwork.net",
+                            "deffile=data/threeImageNetwork/autoRegTemplate.def",
+                            "flatfile=" + flatFilePath,
+                            "onet=" + outNetPath,
+                            "outputfailed=no",
+                            "points=nonignored" };
+  QVector<QString> argsD = { "fromlist=" + cubeListFile,
+                            "cnet=data/threeImageNetwork/controlnetwork.net",
+                            "deffile=data/threeImageNetwork/autoRegTemplate.def",
+                            "flatfile=" + flatFilePath,
+                            "onet=" + outNetPath,
+                            "outputignored=no",
+                            "points=nonignored" };
+  QVector<QString> argsE = { "fromlist=" + cubeListFile,
+                            "cnet=data/threeImageNetwork/controlnetwork.net",
+                            "deffile=data/threeImageNetwork/autoRegTemplate.def",
+                            "flatfile=" + flatFilePath,
+                            "onet=" + outNetPath,
+                            "outputignored=no",
+                            "points=ignored" };
```

Comment:
some of these options look the same? If they are redundant, I would just remove them.